table/src/main/java/co/table/sdk/android/chat/JitsiVideoActivity.kt

```diff
@@ -0,0 +1,57 @@
+package co.table.sdk.android.chat
+
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+
+import co.table.sdk.android.R
+import co.table.sdk.android.constants.Constants
+import org.jitsi.meet.sdk.*
+import java.net.MalformedURLException
+import java.net.URL
+
+class JitsiVideoActivity : AppCompatActivity() {
+    var firstLoad: Boolean = true
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_jitsi_video)
+
+        // Initialize default options for Jitsi Meet conferences.
+        val serverURL: URL
+        val tenant: String = intent.getStringExtra(Constants.B_TENANT)!!
+        val roomID: String = intent.getStringExtra(Constants.B_ROOMID)!!
+        val jwt: String = intent.getStringExtra(Constants.B_JWT)!!
```

Comment:
Maybe we could be a little more defensive rather than the !! operator?

Reply:
I copied over the method from the opentok video activity, I kept it because there shouldn't be a case where we launch this activity without the extras so I thought it is fine in this case.